games-action/dave_gnukem/dave_gnukem-1.0.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="2D platform shooter inspired by Duke Nukem 1"
+DESCRIPTION="Dake Gnukem is a retro-style 2D scrolling platform shooter \
+similar to, and inspired by, Duke Nukem 1.
+
+The year is \$CURRENTYEAR+8. 
+An evil genius, Dr Proetton, has been hired by the CIA \
+to infect the world's computers with a virus called SystemD, crippling them. \
+Only you can stop him. You must find the floppy disk with the \
+Devuan Antivirus on it, and install it on the master computer, \
+which is hidden in Vault7.
+TL;DR: Shoot anything that moves!
+
+Controls:
+Left/right -> Move left/right
+Ctrl -> Jump
+Alt -> Shoot
+Up -> Action key (open doors, use teleporters, etc.)
+Escape -> Open the menu
+"
+
+HOMEPAGE="http://djoffe.com/gnukem/"
+REVISION="1"
+COPYRIGHT="1998-2019 David Joffe"
+LICENSE="GNU GPL v2"
+SOURCE_DIR="dave_gnukem-$portVersion"
+
+SOURCE_URI="https://github.com/davidjoffe/dave_gnukem/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="b4180e94855fe1ad5eb29c4ec9c249f8e23fbd7a602f5c821dfe68c23325ad0d"
+
+SOURCE_URI_2="https://github.com/davidjoffe/gnukem_data/archive/v$portVersion.tar.gz"
```

Comment:
The data repository doesn't mention any license… I suppose it's the same, but it'd be nice to create a ticket there to propose to explicitly put a license.

Reply:
https://github.com/davidjoffe/gnukem_data/issues/1
I've created an issue there asking them to do it.

Reply:
I think they covered the license for those in the link I posted earlier: https://github.com/davidjoffe/dave_gnukem#license-and-additional-credits

Reply:
That's true. I'll remove the issue. Ooops.